dropwizard/service/src/test/java/org/apache/polaris/service/dropwizard/PolarisApplicationIntegrationTest.java

```diff
@@ -712,7 +710,10 @@ public void testRequestHeaderTooLarge() {
 
   @Test
   public void testRequestBodyTooLarge() {
-    // The size is set to be higher than the limit in polaris-server-integrationtest.yml
+    // The behaviour in case of large requests depends on the specific server configuration.
+    // This test assumes that the server under test is configured to deny requests larger than
+    // 1000000 bytes. The test payload below assumes UTF8 encoding of ASCII charts plus a bit of
```

Comment:
We don't need to do this here, but I wonder if we could somehow add a check for the limit to this test.